Flaky test: TestDiff (possibly flaky?)

Seen failing on https://ci-next.docker.com/public/job/moby/job/PR-42451/5/execution/node/300/log/?consoleFull
[amd64-bundles.tar.gz](https://github.com/moby/moby/files/6602557/amd64-bundles.tar.gz)

```
=== RUN   TestDiff
--- FAIL: TestDiff (1.03s)
    diff_test.go:42: assertion failed: 
        --- expected
        +++ items
        {[]container.ContainerChangeResponseItem}:
        	-: []container.ContainerChangeResponseItem{{Kind: 0x01, Path: "/foo"}, {Kind: 0x01, Path: "/foo/bar"}}
        	+: []container.ContainerChangeResponseItem(nil)
```

--- a/integration/container/diff_test.go
+++ b/integration/container/diff_test.go
@@ -24,7 +24,7 @@ func TestDiff(t *testing.T) {
 		{Kind: containertypes.ChangeAdd, Path: "/foo/bar"},
 	}
 
-	poll.WaitOn(t, container.IsStopped(ctx, apiClient, cID))
+	poll.WaitOn(t, container.IsInState(ctx, apiClient, cID, containertypes.StateExited), poll.WithTimeout(60*time.Second))
 	result, err := apiClient.ContainerDiff(ctx, cID, client.ContainerDiffOptions{})
 	assert.NilError(t, err)
 	assert.DeepEqual(t, expected, result.Changes)

